ChaosMod/Util/OptionsFile.h

```diff
@@ -93,4 +114,20 @@ class OptionsFile
 
 		return defaultValue;
 	}
+
+	inline void SetValueString(const std::string& szKey, const std::string& value)
+	{
+		if (m_Options.contains(szKey))
+		{
+			m_Options[szKey] = value;
+		}
+		else
+		{
+			m_Options.emplace(szKey, value);
+		}
+	}
+	template <typename T> inline void SetValue(const std::string& szKey, T value)
+	{
+		SetValueString(szKey, std::to_string(value));
+	}
```

Comment:
```suggestion
	inline void SetValueString(const std::string& key, const std::string& value)
	{
		if (m_Options.contains((key))
		{
			m_Options[key] = value;
		}
		else
		{
			m_Options.emplace(key, value);
		}
	}
	template <typename T> inline void SetValue(const std::string& key, T value)
	{
		SetValueString(key, std::to_string(value));
	}
```
Hungarian notation bad